From 403bf35fe952ed1cc457d2625b02256074d2c074 Mon Sep 17 00:00:00 2001
From: Ionut Vicovan <Ionut.Vicovan@nxp.com>
Date: Wed, 3 Jun 2020 01:00:15 +0300
Subject: [PATCH] Fix fiptool build error.

Based on meta-freescale commit 8ca3647a32841fc72cf097de933cb188e39b51db.
(https://git.yoctoproject.org/cgit/cgit.cgi/meta-freescale/commit/?id=8ca3647a32841fc72cf097de933cb188e39b51db)

Signed-off-by: Ionut Vicovan <Ionut.Vicovan@nxp.com>
---
 tools/fiptool/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/tools/fiptool/Makefile b/tools/fiptool/Makefile
index ef35014..8965022 100644
--- a/tools/fiptool/Makefile
+++ b/tools/fiptool/Makefile
@@ -19,7 +19,8 @@ ifeq (${DEBUG},1)
 else
   HOSTCCFLAGS += -O2
 endif
-LDLIBS := -lcrypto
+
+LDLIBS := -Wl,-rpath=${LIBPATH}  -L${LIBPATH} -lcrypto
 
 ifeq (${V},0)
   Q := @
-- 
2.7.4

